src/api/roles/coach/coach.py

```
@@ -616,3 +674,316 @@ def get_coach_earnings(
+
+    accepted = db.exec(
+        select(ClientCoachRequest).where(
+            ClientCoachRequest.client_id == client_id,
+            ClientCoachRequest.coach_id == coach_id,
+        )
+    ).first()
+    if accepted:
+        rel = db.exec(
+            select(ClientCoachRelationship).where(
+                ClientCoachRelationship.request_id == accepted.id,
+                ClientCoachRelationship.is_active == True,
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `ClientCoachRelationship.is_active:` for truth checks

Replace with `ClientCoachRelationship.is_active`

(E712)